internal/resource_backup.go

```diff
@@ -86,6 +86,11 @@ func resourceBackup() *schema.Resource {
 				Description: "Oasis Backup Resource Backup Policy ID field",
 				Optional:    true,
 			},
+			backupRegionIDFieldName: {
```

Comment:
`CreateContext` does not need to pass this value, it will be computed for non-multi-region backups.
Just that read context has it, I have added it here.